src/library_memfs.js

```diff
@@ -7,7 +7,7 @@
 addToLibrary({
   $MEMFS__deps: ['$FS', '$mmapAlloc'],
 #if !ASSERTIONS
-  $MEMFS__postset: `
+  $MEMFS__preset: `
```

Comment:
We sometimes thrown `MEMFS.doesNotExistError` inside `FS.staticInit()` which happens before we get to `MEMFS__postset`.